libminifi/include/utils/StringUtils.h

```diff
@@ -75,7 +75,12 @@ class StringUtils {
 
   static std::string toLower(std::string_view str);
 
-  // Trim String utils
+  /**
+   * Strips the line ending (\n or \r\n) from the end of the input line.
+   * @param input_line
+   * @return (stripped line, line ending) pair
+   */
+  static std::pair<std::string, std::string> chomp(const std::string& input_line);
```

Comment:
we seem to migrate our StringUtils to operate on `std::string_view` wherever possible, we might benefit from writing the new ones on the same note

Reply:
That would require modifying `endsWith()` (and, for consistency, `endsWithIgnoreCase()` and `startsWith()`), which would introduce more conflicts with #1168.  If #1168 is merged before this, and it includes the change to `endsWith()`, then I will change `chomp()` similarly.